tools/migrations/add-docker-index.js

```diff
@@ -70,18 +47,10 @@ var BUCKET = {
 };
 
 function getMorayClient(callback) {
-    var client = moray.createClient({
-        connectTimeout: config.moray.connectTimeout || 200,
-        host: config.moray.host,
-        port: config.moray.port,
-        log: log,
-        reconnect: true,
-        retry: (config.moray.retry === false ? false : {
-            retries: Infinity,
-            minTimeout: 1000,
-            maxTimeout: 16000
-        })
-    });
+    var morayConfig = config.moray;
+
+    morayConfig.log = log;
+    var client = moray.createClient(morayConfig);
```

Comment:
raised the question w/ @misterdjules: does the config always contain all the moray parameters we want here?